### PR TITLE
expand the SSP notes

### LINKED ISSUE #228: Update documentation for new and improved LATO 90-day process
Documenting new process for 90 day LATOs for Jez

- [ ] do basic system categorization work (essentially the README + SSP YAML - system name, impact level, POCs, blah blah)
- [ ] system / data flow diagrams(s) optional but desired!
- [ ] articulate summary of system (should be same as project README) and end points for scanning in GSA IT ATO project plan
- [ ] fill out Rules of Engagement for scanning and pen test
- [ ] send to Infrastructure Lead for approval, Infra Lead fills out final GSA IT ATO project plan and sends to Infrastructure and the ISSO

### PR DESCRIPTION
**Pull request by afeld**
Add more guidance about how to fill it out, and move the list from the checklist to the SSP page.

Update: Closes #228.

### CODE REVIEW

**Comment by afeld** [comment]
Bump! Would love a review by someone who isn't @hollyallen 😉

File: _includes/checklist.md [hunk 1]
```diff
@@ -59,16 +59,12 @@ Everything in this section needs to be completed before the project will be sche
...
-- [ ] Add an [`.about.yml`](https://github.com/18F/about_yml) for the main repository
-- [ ] [Update relevant documentation](https://pages.18f.gov/before-you-ship/ato/tips/), primarily the README
-- [ ] Fill out the System Security Plan (SSP)
-    * Work with your Infrastructure Lead on this.
-    * Totally fine for this to be a rough first pass.
-    * Your Infrastructure Lead will tell you if you should [do this via Compliance Masonry](https://github.com/18F/cg-compliance#starting-ato-documentation-for-cloudgov-applications)
+- [ ] Fill out the Project Plan.
+- [ ] Add an [`.about.yml`](https://github.com/18F/about_yml) for the main repository.
+- [ ] [Update relevant documentation](https://pages.18f.gov/before-you-ship/ato/tips/), primarily the README.
+- [ ] [Fill out the System Security Plan (SSP).](https://pages.18f.gov/before-you-ship/ato/ssp/)
```
Comment (theresaanna) [inline _includes/checklist.md]: Do we want to note that 90 days don't need the SSP?

Comment (afeld) [inline _includes/checklist.md]: Done!

Comment (afeld) [inline _includes/checklist.md]: /cc #228

**Pull request merged**